### ISSUE TITLE
Completion proposal popup window ?

### ISSUE THREAD

**Issue by ValeriyTitov** [bonecode/BCEditor#76]
As i understood, this is about this window in SSMS ?
If it so, can you provide "Hello world" example of using it ?

![02](https://user-images.githubusercontent.com/34241725/44836695-55131b00-ac40-11e8-8d24-fa01e495bee6.png)

**Comment by NilsHoyer** [comment]
You are right, an example would be nice. But the question is: Should I go ahead working on the BCEditor project. There are not many feedback. And for my own usage, I have what I want.

Btw, I use the CompletionProposal Window for SQL code completion - on every SQL command, on every place there are lists of valid words. But this is a lot of work - I have to parse and understand the SQL commands.

Than I use BCEditor.OnCompletionProposalShow event to fill the BCEditor.CompletionProposal.Column[0].Items list. That list will be shown on my application.

Than I added a second column with a small hint what kind of word it is: Keyword, Function, Field, Constante, ...

... like mentioned: It's a lot of work to fill the Items List with the correct words. But this can't do BCEditor by itself - it depends on the context and used highlighter. (There are a lot of differences between MySQL and Oracal SQL for example.)

**Comment by ValeriyTitov** [comment]
Im planning to fill this popup with reserved words of current syntax (loaded with Json) filtered by string what user currently typing. 
But once again, this is seperate task.

**Comment by NilsHoyer** [comment]
By default cpoAddHighlighterKeywords is set in CompletionProposal.Options. So the Keywords from the highlighter should be filled automatically.
